aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+const Error = ({ children, ...rest }) => {
+  return (
+    <Box direction="row" {...rest}>
+      <Box flex={false} margin={{ top: 'xxsmall', right: 'xxsmall' }}>
+        <CircleAlert size="small" />
+      </Box>
+      <Text size="xsmall">{children}</Text>
+    </Box>
+  );
+};
+
+Error.propTypes = {
+  children: PropTypes.object,
+};
+
+export const RequiredFieldsExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    name: 'Enduro',
+    superPower: 'Sky Runner',
+    nemesis: true,
+    email: 'enduro@skyrunner.io.com',
+    weakness: 'PB & J',
+  });
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Text size="xxlarge" weight="bold">
+            Form Header
+          </Text>
+        </Header>
+        <Box
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            messages={{
+              required: (
+                // need to define background otherwise
+                // inherits validation-critical background
+                <Error background="background-front">
```

Comment:
Seems like all of the backgrounds here should be omitted since they are used the most and the only instance supplying a background color should be the bottom form error message.
```suggestion
                <Error >
```

Reply:
without defining the background it gets the formfield validation-error color